pkg/chunk/storage/factory.go

```diff
@@ -267,7 +270,7 @@ func newChunkClientFromStore(store chunk.ObjectClient, err error) (chunk.Client,
 }
 
 // NewTableClient makes a new table client based on the configuration.
-func NewTableClient(name string, cfg Config) (chunk.TableClient, error) {
+func NewTableClient(name string, cfg Config, registerer prometheus.Registerer) (chunk.TableClient, error) {
```

Comment:
This is called in a couple of places in `modules.go`. Shouldn't we wrap the registered there as well to add the `component` label?

Reply:
We don't need to because we currently only manage tables for current periodic config but I have changed it anyways.